bin/capsule-certs-generate

```diff
@@ -45,11 +45,20 @@ CONFIG_CONTENT = <<EOF
 EOF
 
 config_file = temp_file('capsule-certs', CONFIG_CONTENT)
+
+# generate our parser cache and add it to the config file
+parser_cache = temp_file('capsule-certs.parser_cache', `kafo-export-params -c #{config_file.path} -f parsercache --no-parser-cache`)
```

Comment:
Correct -- given our previous reasons we were avoiding packaging puppet-strings.